### DISCUSSION TITLE
Have uptime kuma status page bypass authelia

### DISCUSSION THREAD

**Discussion by mdhwoods** [authelia/authelia#8276] [Q&A] [answered]
Want to protect uptime kuma behind authelia but want to have access to the status page without authentication.  i have tried several options with none being effective. Anyone have any guidance?

TIA

**Comment by james-d-elliott** [comment, marked as answer]
You can use basic auth and use the `one_factor` policy for a specific user or use [bearer auth with the client credentials grant](https://www.authelia.com/integration/openid-connect/uptime-kuma/)
